src/Chapter2.hs

```diff
@@ -336,7 +349,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList i j a
+    | i <= j = take (j-i+1) (drop i a)
```

Comment:
Nice catch on the condition! 👍🏼 However, note that on the negative input it could show wrong results.